src/main/java/com/google/sps/tool/EntityTestingTool.java

```diff
@@ -0,0 +1,58 @@
+package com.google.sps.tool;
+
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Key;
+import com.google.appengine.api.datastore.KeyFactory;
+
+import java.util.List;
+import java.util.ArrayList;
+import com.google.sps.data.Card;
+import com.google.sps.data.Folder;
+
+public class EntityTestingTool {
+
+  public static boolean checkForNoNullValues(Entity card) {
```

Comment:
Code isn't used.

Reply:
This is used in UserCardsServletTest.java in the above branch. Yeah -- sorry, I currently have some sync problem in the beginning.